build.gradle

```diff
@@ -125,45 +125,21 @@ dependencyCheck {
 
 repositories {
   jcenter()
-  // remove once spring sorts out their cloud artifact to proper version
+
   maven {
-    url "https://repo.spring.io/libs-milestone"
+    url 'https://dl.bintray.com/palantir/releases' // docker-compose-rule is published on bintray
```

Comment:
I think we could remove this docker compose rule and Azurite test later. Not sure what benefit we are getting from Azurite? Not part of this PR just commenting here so we don't miss it later.

Reply:
would be good. locally integration tests did not pass 😕